H/H3/build_tarballs.jl

```diff
@@ -3,11 +3,11 @@
 using BinaryBuilder, Pkg
 
 name = "H3"
-version = v"4.1.0"
+version = v"4.2.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/uber/h3.git", "5c91149104ac02c4f06faa4fc557e69cf6b131ef")
+    GitSource("https://github.com/uber/h3.git", "42450ce2666ce068a72a7cbc47d1b5157f67e720")
```

Comment:
That's not v4.2.0, it's a random commit after v4.2.0.  In general I'd feel much more comfortable with building v4.2.0 + a single patch, that said, [changes after v4.2.0](https://github.com/uber/h3/compare/v4.2.0...42450ce2666ce068a72a7cbc47d1b5157f67e720) are relatively minor (mainly reformatting tests, adjusting CI setup and changing name of variables in source code), so let's do it for this time.

Reply:
We've now released [v4.2.1](https://github.com/uber/h3/releases/tag/v4.2.1) which should include the fixes you need